cli.js

```diff
@@ -58,6 +58,8 @@ if ("--ramification" in cliFlags)
     globalThis.RAMification = true;
 if ("--no-prefetch" in cliFlags)
     globalThis.prefetchResources = false;
+if ("--details" in cliFlags)
```

Comment:
Since this is really just about grouped benchmarks, can we give this flag (and the respective variables) a more descriptive name? Something like `--expand-grouped-workloads` or `--group-details` or so?